parlai/tasks/fromfile/agents.py

```diff
@@ -53,62 +53,7 @@ def __init__(self, opt, shared=None):
         
 class ParlaiformatTeacher(FixedDialogTeacher):
```

Comment:
would this class now subclass `ParlAIDialogTeacher` rather than `FixedDialogTeacher`?